webserver/templates/datasets/evaluate.html

```diff
@@ -21,6 +21,10 @@ <h2 class="page-title">Evaluate dataset "{{ dataset['name'] }}"</h2>
         <div class="col-sm-4">{{ form.filter_type(class="form-control", required="required") }}</div>
       </div>
       <div class="form-group">
+        <label class="col-sm-2 control-label">{{ form.evaluation_location.label.text }}</label>
```

Comment:
we talked briefly in IRC about having a "feature flag" so that we can merge this patch and release it but disable this form element.

You can have a section in `config.py` and have a variable called `FEATURE_something`. If it is set to `False`, don't show this option here, and set `eval_location` to `local` always. Make sure that it is easy to remove this setting when we make the feature permanent (hopefully we only want 2 or 3 if statements if possible)